### PR TITLE
Add TN Events

### PR DESCRIPTION
**Pull request by askpatrickw**
Tennessee State Motto: Agriculture and Commerce

### CODE REVIEW

**Comment by askpatrickw** [comment]
This isn't ready, but I can't put to Draft.
Missed .absentee ... twice. :-/

**Comment by askpatrickw** [comment]
NVM, it changed to draft, the dialog just didn't go away when I clicked it.

File: states/tennessee/elections.toml [hunk 1]
```diff
@@ -8,7 +8,68 @@ date = 2020-08-06
...
+    "https://sos.tn.gov/products/elections/2020-election-calendar",
+    "https://sos.tn.gov/products/elections/how-do-i-register-vote"
+]
+
+[20201103.absentee.application]
+receive_start = 2020-08-05
+received_by = 2020-10-27
+sources = ["https://sos.tn.gov/products/elections/2020-election-calendar"]
+
+[20201103.asbsentee]
```
Comment (tannewt) [inline states/tennessee/elections.toml]:
```suggestion
[20201103.absentee]
```

File: states/tennessee/elections.toml [hunk 2]
```diff
@@ -8,7 +8,68 @@ date = 2020-08-06
...
+    "https://sos.tn.gov/products/elections/2020-election-calendar",
+    "https://sos.tn.gov/products/elections/how-do-i-register-vote"
+]
+
+[20200806.absentee.application]
+receive_start = 2020-05-08
+received_by = 2020-07-30
+sources = ["https://sos.tn.gov/products/elections/2020-election-calendar"]
+
+[20200806.asbsentee]
```
Comment (tannewt) [inline states/tennessee/elections.toml]:
```suggestion
[20200806.absentee]
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Two typos. Dates look good!

**Comment by askpatrickw** [comment]
Thanks @tannewt  !

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**